tests/recorded/reorder_none.txt

```diff
@@ -0,0 +1,39 @@
+def f(x: Union[int, None, str] = 3):
+    pass
+
+def g(x: Literal[None, 1] = None):
+    pass
+
+Union[int, None, str]
+Literal[None, 1]
+
+================================================================================
+
+def f(
+    x: Union[
+        int,
+        str,
+        None,
+    ] = 3
```

Comment:
I'm not a fan of how this splits out over multiple lines - can you make sure that the last subscript_slice does *not* have a comma?  That way Black will keep it on one line, at least if it fits.

Reply:
Ah, I thought that this was something that Black just did by itself. I fixed this in the latest commit.